### PR TITLE
Add semanticdbEnable command for easier quick installation.

### PR DESCRIPTION
**Pull request by olafurpg**
I kept everything in the same file to retain the single file copy
installation step in BETA.md.

### CODE REVIEW

File: BETA.md [hunk 1]
```diff
@@ -32,14 +32,24 @@ From the repo root run `sbt publishLocal`
...
+There are two alternative ways to install `semanticdb-scalac`.
+
+The first option is to enable `semanticdb-scalac` permanently for your project in `build.sbt` with:
+
+```scala
+libraryDependencies += compilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.2" cross CrossVersion.full)
+scalacOptions += "-Yrangepos"
+```
+
+The second option is to enable `semanticdb-scalac` only for an active sbt session by running `semanticdbEnable` from the sbt shell.
```
Comment (cquiroz) [inline BETA.md]: I was exactly looking for this, but can this be enable without touching any of the `sbt` files?

Comment (olafurpg) [reply BETA.md]: Yes, this works without touching any sbt files.

**Comment by cquiroz** [comment]
I tested this copying the new file to my `~/.sbt/` files but I don't seem to get the command

```
root> semanticdbEnable
[error] Not a valid command: semanticdbEnable
[error] Not a valid project ID: semanticdbEnable
```

I get the command for projects that already have the semanticdb plugin. Any ideas on how to debug this?

**Comment by olafurpg** [comment]
@cquiroz One way I typically debug issues like that is to run `plugins` and look for "ScalametaLanguageServerPlugin". If it's not in the list then the plugin is not loaded. Where did you put the file? It should be in `~/.sbt/0.13/plugins/ScalametaLanguageServerPlugin.scala`

**Comment by olafurpg** [comment]
Note that the location is different for sbt 1.0 https://github.com/scalameta/language-server/blob/master/BETA.md#step-1---sbt-plugin

**Comment by cquiroz** [comment]
It seems there are wrong instructions. with the right location for sbt 0.13 it works fine, see #145

**Pull request merged**